packages/teleport/src/Kubes/KubeList/KubeList.tsx

```diff
@@ -79,11 +81,43 @@ function KubeList(props: Props) {
           cell={<TextCell />}
         />
         <Column header={<Cell>Labels</Cell>} cell={<LabelCell />} />
+        <Column
+          header={<Cell />}
+          cell={
+            <ActionCell
+              onViewConnect={(kubeName: string) => setKubeName(kubeName)}
+            />
+          }
+        />
       </StyledTable>
+      {kubeName && (
```

Comment:
hmm i wonder how we can rename `kubeName` so it gives more context... maybe `kubeConnectName`?

Reply:
I agree `kubeConnectName` gives it more context.